frontend/pages/events.tsx

```diff
@@ -39,6 +56,151 @@ const CardList = styled.div`
   }
 `
 
+const localizer = momentLocalizer(moment)
+
+const CalendarEvent = ({
+  event: { resource },
+}: {
+  event: { resource: ClubEvent }
+}) => {
+  return (
+    <>
+      {resource.name} - {resource.club_name}
+    </>
+  )
+}
+
+enum EventsViewOption {
+  LIST = 'LIST',
+  CALENDAR = 'CALENDAR',
+}
+
+enum CalendarNavigation {
+  PREVIOUS = 'PREV',
+  NEXT = 'NEXT',
+  TODAY = 'TODAY',
+  DATE = 'DATE',
+}
+
+enum CalendarView {
+  MONTH = 'month',
+  WEEK = 'week',
+  WORK_WEEK = 'work_week',
+  DAY = 'day',
+  AGENDA = 'agenda',
+}
+interface CalendarHeaderProps {
+  date: Date
+  label: string
+  onNavigate(action: CalendarNavigation, newDate?: Date): void
+  onView(view: CalendarView): void
+  view: string
+  views: CalendarView[]
+}
+
+const StyledHeader = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: 20px 0;
+  .tools {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+    .view-label {
+      font-size: 18px;
+    }
+    & > *:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+`
+
+const ViewContext = createContext<
+  [option: EventsViewOption, setOption?: (args?: any) => any]
+>([EventsViewOption.CALENDAR])
+
+const EventsViewSwitcher = ({ viewOption, setViewOption }) => (
+  <SegmentedButton
+    buttons={[
+      {
+        key: 'segmentbutton-list',
+        icon: 'grid',
+        selected: viewOption === EventsViewOption.LIST,
+        onClick: () => {
+          setViewOption(EventsViewOption.LIST)
+        },
+      },
+      {
+        key: 'segmentbutton-calendar',
+        icon: 'calendar',
+        selected: viewOption === EventsViewOption.CALENDAR,
+        onClick: () => {
+          setViewOption(EventsViewOption.CALENDAR)
+        },
+      },
+    ]}
+  />
+)
+
+const CalendarHeader = ({
+  // date,
```

Comment:
@esinx If this won't be used in the near future, it'll be a good idea to remove it for now (the comment and the type annotation) to avoid confusion later on.

Reply:
Yup! Seems like it shouldn't be there.